File: Documentation Code Snippets/ScanbotSDK Examples/EHICScanner/DetectionOnImage/Objc/EhicDetectionOnImage.m
//
//  EhicDetectionOnImage.m
//  ScanbotSDK Examples
//
//  Created by Rana Sohaib on 27.03.23.
//

#import "EhicDetectionOnImage.h"
@import ScanbotSDK;

@implementation EhicDetectionOnImage

- (void) detectEhicOnImage {
    
    // Image containing EU Health Insurance Card.
    UIImage *image = [UIImage imageNamed:@"ehicImage"];
    
    // Creates an instance of `SBSDKHealthInsuranceCardRecognizer`.
    SBSDKHealthInsuranceCardRecognizer *detector = [[SBSDKHealthInsuranceCardRecognizer alloc] init];
    
    // Returns the result after running detector on the image.
    SBSDKHealthInsuranceCardRecognitionResult *result = [detector recognizeOnStillImage:image];
}

@end

File: Documentation Code Snippets/ScanbotSDK Examples/EHICScanner/DetectionOnImage/Swift/EhicDetectionOnImage.swift
//
//  EhicDetectionOnImage.swift
//  ScanbotSDK Examples
//
//  Created by Rana Sohaib on 27.03.23.
//

import Foundation
import ScanbotSDK
    
func detectEhicOnImage() {
    
    // Image containing EU Health Insurance Card.
    guard let image = UIImage(named: "ehicImage") else { return }
    
    // Creates an instance of `SBSDKHealthInsuranceCardRecognizer`.
    let detector = SBSDKHealthInsuranceCardRecognizer()
    
    // Returns the result after running detector on the image.
    let result = detector.recognize(onStillImage: image)
}
